interface/src/app/gene/network/network.tpl.html

```diff
@@ -0,0 +1,15 @@
+<h1>ADAGE: Gene-Gene Network</h1>
+
+<div ng-hide="ctrl.statusMessage">
```

Comment:
Nice debugging on issue #25! Messing around with the DOM gets extra tricky when you're integrating Angular with external libraries such as `d3` and `vega`.